src/category_theory/fully_faithful.lean

```diff
@@ -81,6 +80,68 @@ variables (F : C ⥤ D) (G : D ⥤ E)
 
 instance faithful.comp [faithful F] [faithful G] : faithful (F ⋙ G) :=
 { injectivity' := λ _ _ _ _ p, F.injectivity (G.injectivity p) }
+
+lemma faithful.of_comp [faithful $ F ⋙ G] : faithful F :=
+{ injectivity' := λ X Y, (F ⋙ G).injectivity.of_comp }
+
+variables {F G}
+
+lemma faithful.of_comp_eq {H : C ⥤ E} [ℋ : faithful H] (h : F ⋙ G = H) : faithful F :=
+@faithful.of_comp _ _ _ _ _ _ F G (h.symm ▸ ℋ)
+
+alias faithful.of_comp_eq ← eq.faithful_of_comp
+
+variables (F G)
+
+protected def faithful.div (F : C ⥤ E) [faithful F] (G : D ⥤ E) [faithful G]
```

Comment:
This (and `faithful.div_comp`) don't actually use `[faithful F]`. (Indeed, `#sanity_check` detects this.)

Reply:
What is `div` actually short for here? Perhaps we need a better name here.

Reply:
It's short for `divide`. I'll remove the `faithful F` argument.

Reply:
Added a comment, removed the unused argument. Feel free to rename.